features/networking/multus.feature

```diff
@@ -1133,19 +1133,18 @@ Feature: Multus-CNI related scenarios
   Scenario: Create pod with Multus ipvlan CNI plugin	
     # Make sure that the multus is enabled
     Given the multus is enabled on the cluster
-    And I store all worker nodes to the :nodes clipboard
     And the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
     #Storing default interface mac address for comparison later with pods macs
-    Given I use the "<%= cb.nodes[0].name %>" node
-    And I run commands on the host:
+    Given I run commands on the host:
```

Comment:
Before this line, should script still need the step of Given I use the "<%= cb.target_node %>" node? This step will get MAC from the host, and the rest script will check pod's MAC will be same as the MAC from the cb.target_node. Hopefully the host here is just the cb.target_node, otherwise the MAC checking will fail.

Reply:
@weliang1 This is fine. In this case ```cb.nodes[0].name``` will be the host only. If you are using these steps back to back without changing any course, your ```node.name``` will always be your host. Fore more info
Ref: https://github.com/openshift/verification-tests/blob/master/lib/openshift/node.rb#L27